contrib/epee/include/net/http_client.h

```diff
@@ -527,7 +538,11 @@ using namespace std;
 				}
 				CHECK_AND_ASSERT_MES(m_len_in_remain >= recv_buff.size(), false, "m_len_in_remain >= recv_buff.size()");
 				m_len_in_remain -= recv_buff.size();
-				m_pcontent_encoding_handler->update_in(recv_buff);
+				if (!m_pcontent_encoding_handler->update_in(recv_buff))
```

Comment:
If you mean "wasn't it bailing out on false", it wasn't. But this so obvious that I'm probably missing your point :)